stock_shift_residual_qty/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,10 @@
+Typical use case:
+
+- you deliver products with lots and expiry dates to your warehouse coming from production.
+- you only send round quantity depending of your packaging (i.e. pallets)
+- you don't want Odoo propose in the next outgoing picking, a quantity lower than needed by your conditioning. 
```

Comment:
you don't want for the next outgoing picking, that Odoo proposes a lower quantity than needed by your conditioning.